### PR TITLE
Shponvader nor more based on connector_locomotivecms

### PR DESCRIPTION
**Pull request by lmignon**

### CODE REVIEW

File: shopinvader/migrations/10.0.2.0.0/post_migrate.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from openupgradelib import openupgrade
+
+
+def migrate(cr, version):
+    """Update database from previous versions, after updating module."""
+    openupgrade.update_module_moved_fields(
+        cr=cr,
+        model='res.partner',
+        moved_fields=['lang'],
+        old_module='brx_account',
```
Comment (sebastienbeau) [inline shopinvader/migrations/10.0.2.0.0/post_migrate.py]: copy/paste error ;)

File: shopinvader/tests/__init__.py [hunk 2]
```diff
@@ -1,12 +1,3 @@
 # -*- coding: utf-8 -*-
 
-from . import test_notification
```
Comment (sebastienbeau) [inline shopinvader/tests/__init__.py]: oups ! c'est pour ça que c'est tout vert ;)

**Review by sebastienbeau — COMMENTED** [review]
Good job !!

**Comment by rvalyi** [comment]
Just to understand, what is the purpose? Be able to have other web stack than Locomotive? Like Django or  other CMS's ?

**Comment by sebastienbeau** [comment]
Yes in some special case, when a customer already have a cms maybe he can integrate the same logic in order to interact with odoo with the same module as the module used for shopinvader_locomotive.
The more generic we are the most huge community we can bring on this project

We also need to split more the shopinvader module (base_invader without cart logic). So if you just want to add a form on a wordpress website you can use base_invader + lead_invader

**Comment by rvalyi** [comment]
sounds good. thanks. BTW Paypal module is coming...

**Pull request merged**